eng/common/TestResources/New-TestResources.ps1

```diff
@@ -286,7 +286,21 @@ foreach ($templateFile in $templateFiles) {
         Log "Persist the following environment variables based on your detected shell ($shell):`n"
     }
 
-    $deploymentOutputs = @{}
+    $serviceDirectoryPrefix = $ServiceDirectory.ToUpperInvariant()
```

Comment:
Future proofing to make sure we can have multiple parallel deployments and they won't clash.

Reply:
This can be a relative or absolute path as well. You want `$serviceName`.

Reply:
Good point.